ocean_provider/run.py

```diff
@@ -25,11 +25,30 @@ def get_version():
 
 @app.route("/")
 def version():
+    """
+    Contains the provider data for an user:
+        - software;
+        - version;
+        - network url;
+        - provider address;
+        - services endpoints, which has all
+        the existing endpoints from route.py.
+    """
     info = dict()
     info['software'] = Metadata.TITLE
     info['version'] = get_version()
     info['network-url'] = config.network_url
     info['provider-address'] = get_provider_wallet().address
+    info['servicesEndpoints'] = {
```

Comment:
@ssallam @alexcos20 I was wondering about the logic here. If we want these to be constantly filled with all services routes, without manual adding every time, we can do that. However, it seems like the services are customly named: e.g. 'access' instead of 'download' etc.

Are you ok with manual adding to this dictionary, whenever new services are added, or should we work on something that automatically takes all the services available? E.g.:

`{url.endpoint.replace('services.', ''): '%s' % url for url in app.url_map.iter_rules() if url.endpoint.startswith('services.')}`

Reply:
I also had this question and I understood that I can create a dictionary for manual adding the services, but if there are more suggestions about this, please let me know and I will refactor it. @ssallam & @alexcos20 .

Reply:
There is always the option of having a naming override dictionary. Consider all keys equivalent to the route names unless overridden. E.g.
<img width="848" alt="Screenshot 2021-01-09 at 12 23 44" src="https://user-images.githubusercontent.com/4101015/104089208-8f137380-5275-11eb-8694-e12c79e55599.png">

Reply:
Automatically generating the service endpoints dict makes sense and having an override for specific names would be great. I suggest renaming the functions if necessary so that the names don't need override.